test/functional/rpc_scanblocks.py

```
@@ -0,0 +1,129 @@
+        # make sure the blockhash is present when using the first mined block as start_height
+        assert(blockhash in node.scanblocks(
+            "start", [{"desc": f"pkh({parent_key}/*)", "range": [0, 100]}], height)['relevant_blocks'])
+
+        # check that false-positives are included in the result now; note that
+        # finding a false-positive at runtime would take too long, hence we simply
+        # use a pre-calculated one that collides with the regtest genesis block's
+        # coinbase output and verify that their BIP158 ranged hashes match
+        genesis_blockhash = node.getblockhash(0)
+        genesis_spks = bip158_relevant_scriptpubkeys(node, genesis_blockhash)
+        assert_equal(len(genesis_spks), 1)
+        genesis_coinbase_spk = list(genesis_spks)[0]
```

Ruff (0.14.5)

Prefer `next(iter(genesis_spks))` over single element slice

Replace with `next(iter(genesis_spks))`

(RUF015)